src/main/resources/application.properties

```diff
@@ -4,5 +4,5 @@ spring.datasource.username=
 spring.datasource.password=
```

Comment:
also, try figuring out and setting up the env variables to inject passwords and other sensitive data from outside of the application in spring and application.properties as an additional task, would be beneficial, maybe not now but later on